add tc-tag-missing or tc-tag-exists to tag pills including docs

pmario opened the pull request:
This PR fixes #7950

- #7950

- It implements 2 new classes: `tc-tag-missing` and `tc-tag-exists`
- There is a new "example" in the example tiddler
- There is some interactive info in the Tag Macro documentation tiddler to disable the setting, if needed

BurningTreeC commented:
Hello @pmario ,

shouldn't the filter be [<__tag__>!is[shadow]is[missing] ... ?

Jermolene commented on themes/tiddlywiki/vanilla/base.tid:
```diff
@@ -783,6 +783,10 @@ button svg.tc-image-button, button .tc-image-button img {
 	margin-right: 7px;
 }
 
+.tc-tag-missing {
```
I do not think that missing tags should be italicised by default. In many situations, the presence or absence of the tag tiddler is just a consequence of whether the tagged tiddlers have been manually reordered through drag and drop. I do see that some people would want it as an option, but I don't think the core should provide a UI for it as an option.

Jermolene commented on editions/tw5.com/tiddlers/macros/TagMacro.tid:
```diff
@@ -1,17 +1,59 @@
+: This class is defined if a tag does exist as a tiddler. This class has ''no'' default definition
+
+
+!! Disabling CSS classes
```
I understand the intent of this section but I think it complicates the documentation. It's really a helper tool that should exist in its own right

Jermolene reviewed (commented):
Thanks @pmario I like the idea of adding the classes, but I have some concerns about other aspects.

pmario commented:
- Did remove the default CSS setting. 
- Did add simplified docs about how to create the needed stylesheet
- added `<<.from-version` info
- adjusted example info text

saqimtiaz commented on editions/tw5.com/tiddlers/macros/TagMacro.tid:
```diff
@@ -11,7 +11,33 @@ The <<.def tag>> [[macro|Macros]] generates a tag pill for a specified tag. Clic
+<<.from-version "v5.3.4">>
+
+; `tc-tag-missing`
+: This class is defined if a tag does ''not exist'' as a tiddler. By default the format is set to //italic//
```
How is the default italic text if there are no default CSS definitions for it in the core?

pmario replied:
uups. forgot to change that

pmario commented:
@saqimtiaz -- fixed it. There should be a new preview build soon

pmario commented:
@Jermolene -- please review again. The default styling has been removed from the core CSS. It has been replaced with some more documentation, how users can make missing tags "italic". 

See: https://tiddlywiki5-git-fork-pmario-7950-implement-tc-5bb40f-jermolene.vercel.app/#tag%20Macro:%5B%5Btag%20Macro%5D%5D%20%5B%5Btag%20Macro%20(Examples)%5D%5D

Jermolene commented:
Thanks @pmario
